src/panel/MasqStatus.js

```diff
@@ -0,0 +1,89 @@
+/* globals _ */
```

Comment:
What about naming new React components with `.jsx` extension ? This way it can be easier to track which components still need to be migrated, and editors like vscode better handle these files by default

Reply:
Why not. That's the kind of things that is important if we want to share common practices with you :) As it requires a change in webpack config, it should be done in a separate PR though.